packages/clay-autocomplete/package.json

```diff
@@ -31,10 +31,6 @@
 		"@clayui/loading-indicator": "^3.0.0",
 		"fuzzy": "^0.1.3"
 	},
-	"devDependencies": {
-		"@clayui/data-provider": "^3.0.0",
```

Comment:
I noticed that we are removing a lot of devDependencies that are needed for the components storybook. Is this intentional?

Reply:
Yes, these declarations of devDependencies for each package inside our monorepo is unnecessary. We just need these deps to be top level :)